dataeng/resources/warehouse-transforms-ci-dbt.sh

```diff
@@ -11,4 +11,17 @@ dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --prof
 dbt seed --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
 
 dbt run $DBT_RUN_OPTIONS $DBT_RUN_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
-dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
\ No newline at end of file
+dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET && true
```

Comment:
I'm confused here. If the `set` built-in is still in place, then failing commands should cause the script to exit immediately. However, the next line has a check for a nonzero status code. I'm not sure this will work.

Reply:
It works fine as per my testing, we are good on that.